packages/vue-router/src/router.ts

```diff
@@ -265,8 +265,11 @@ export const createIonRouter = (opts: IonicVueRouterOptions, router: Router) =>
         nextRouteInfo.pushedByRoute = leavingRouteInfo?.pathname;
         nextRouteInfo.lastPathname = leavingRouteInfo?.pathname;
       } else if (nextRouteInfo.routerAction === 'push' && isNewTab) {
-        // const lastTabRouteInfo = locationHistory.getCurrentRouteInfoForTab(nextRouteInfo.tab);
-        nextRouteInfo.pushedByRoute = (leavingRouteInfo.pathname !== '') ? leavingRouteInfo.pathname : undefined;
+        if (isLeavingRouteTab) {
```

Comment:
My thoughts behind this were if any last route was a tab, use its pushedByRoute(which would always be a non-tab route) else use the lastPathname. So if the user pushes ion-back-button it would go to a non-tab page, but android device back would just go back -1